workspace/dashboard/src/styles.css

```
@@ -565,14 +903,364 @@ select {
+}
+
+.tab-debug-list {
+  list-style: none;
+  padding: 0;
+  margin: 8px 0 0;
+  font-size: 0.82rem;
+}
+
+.tab-debug-list li {
+  margin-bottom: 4px;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)